homeassistant/components/yolink/strings.json

```diff
@@ -44,6 +44,9 @@
     }
   },
   "entity": {
+    "binary_sensor": {
+      "water_running": { "name": "Water is Running" }
```

Comment:
What is the purpose of the entity? I think we can pick a better name

Reply:
This entity is used to indicate the current water flow status in the pipe.